frontend/util.lua

```diff
@@ -218,13 +226,14 @@ end
 ---- @string Either "modern" for 1h30'10" or "classic" for 1:30:10
 ---- @bool withoutSeconds if true 1h30' or 1h30m, if false 1h30'10" or 1h30m10s
 ---- @bool hmsFormat, modern format only, if true format 1h30m or 1h30m10s
+---- @bool includeDays, if hours>=4 include days in clock string 1d12h10m10s
```

Comment:
>= 24, probably? ;).

Reply:
And may be name the param `withDays` to be consistent with `withoutSeconds` ("include" sounds a bit strange in this context).